### ISSUE TITLE
Register domain link broken

### ISSUE THREAD

**Issue by jpawlowski** [MicrosoftDocs/windows-itpro-docs#582]
The description says:

"To register a domain, follow these steps: [...], administrators can simply [register their domain](https://insider.windows.com/for-business-organization-admin/) and control settings centrally."

However, the link is not working as it would always redirect to the front page of the Insider Program.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bcd37940-410a-d736-1fc4-8122834fa0ee
* Version Independent ID: 58a60d34-d7bc-7ae9-c099-2ce1d3790920
* Content: [Windows Insider Program for Business](https://docs.microsoft.com/en-us/windows/deployment/update/waas-windows-insider-for-business#register-in-the-windows-insider-program-for-business)
* Content Source: [windows/deployment/update/waas-windows-insider-for-business.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/deployment/update/waas-windows-insider-for-business.md)
* Service: **unspecified**
* GitHub Login: @jaimeo
* Microsoft Alias: **jaimeo**

**Comment by jaimeo** [comment]
The link is correct—that is the correct page. There is a section and link on that target page that says “register your organization account.” PM confirms this is right.
vv
J.O.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by jaimeo** [comment]
The link is correct—that is the correct page. There is a section and link on that target page that says “register your organization account.” PM confirms this is right.

**Comment by jpawlowski** [comment]
Thanks, Jaimeo. I saw the link on the page and it would do the same thing, even though I explicitly logged in with my global administrator before. Going directly to that link always redirects me to the front page after asking me to login (which it also does a couple of times in a raw). 

What I did to finally get  to that page:

- login with global admin account
- go to https://insider.windows.com/en-us/for-business-getting-started/
- click on "FIND OUT HOW TO MANAGE INSIDER PREVIEW BUILDS"

It is the very same link behind it but still it would only work when I navigate to the "getting started" from the menu and click the link from there. Using the link from docs.microsoft.com or even copy&paste does not work.
